src/dune_rules/context.ml

```diff
@@ -186,7 +186,7 @@ end = struct
         | Some opam -> (
           let+ version =
             Memo.of_reproducible_fiber
-              (Process.run_capture_line ~display:!Clflags.display Strict opam
+              (Process.run_capture_line ~display:Quiet Strict opam
```

Comment:
We query the version of opam here. This is internal and does not need to be reported to the user.